GVRf/Framework/framework/src/main/java/org/gearvrf/io/GVRTouchPadGestureDetector.java

```diff
@@ -0,0 +1,280 @@
+/************************************************************************************
```

Comment:
I am under the impression we could not include this class in the Framework for some licensing issue. @thomasflynn ?